app/models/course/assessment/question.rb

```diff
@@ -37,11 +37,15 @@ def auto_grader
   #
   # @param [Course::Assessment::Submission] submission The submission which the answer should
   #   belong to.
+  # @param [Course::Assessment::Answer|nil] last_attempt The new answer will be built based on the
```

Comment:
Maybe something like "If last_attempt is given, fields in the new answer will be pre-populated with data from it."